docs/ops/deployment/kubernetes.md

```diff
@@ -143,8 +176,10 @@ data:
     blob.server.port: 6124
     jobmanager.rpc.port: 6123
     taskmanager.rpc.port: 6122
-    jobmanager.memory.process.size: 1472m
-    taskmanager.memory.process.size: 1024m
+    queryable-state.server.ports: 6125
+    jobmanager.memory.process.size: 1600m
```

Comment:
Which of these options are truly required, and which are just for demonstration purposes?

Reply:
I think only `jobmanager.rpc.address` is required